src/platform/Zephyr/BLEManagerImpl.cpp

```diff
@@ -72,7 +72,7 @@ BT_GATT_SERVICE_DEFINE(CHIPoBLE_Service,
                                BT_GATT_PERM_READ | BT_GATT_PERM_WRITE,
```

Comment:
nit: Github does not allow me to comment line 71.. What about removing `BT_GATT_CHRC_WRITE` as Write Request was replaced with Write Command?